draft-irtf-cfrg-frost.md

```diff
@@ -472,13 +460,13 @@ The following function is used to extract a binding factor from a list of bindin
 
   Outputs: A Scalar value.
 
-  Errors: "invalid participant", when the designated participant is not known
+  Errors: "invalid signer", when the designated signer is not known
 
-def binding_factor_for_participant(binding_factor_list, identifier):
+def binding_factor_for_signer(binding_factor_list, identifier):
   for (i, binding_factor) in binding_factor_list:
     if identifier == i:
```

Comment:
Not changed in this PR but the input `i` is described as an `Identifier` but that proper type is not defined in this version of the document